bag_unpacker/src/main/scala/uk/ac/wellcome/platform/archive/bagunpacker/services/BagUnpackerWorkerService.scala

```diff
@@ -25,13 +26,21 @@ class BagUnpackerWorkerService(
     stream.run(processMessage)
 
   def processMessage(
-    unpackBagRequest: UnpackBagRequest
+    unpackBagRequest: UnpackRequest
   ): Future[Unit] =
     outgoingSnsWriter
       .writeMessage(
         BagRequest(
           unpackBagRequest.requestId,
-          unpackBagRequest.bagDestination
+          // TODO: The unpacker will need some more info
+          // namespace/prefix from config.
+          // externalIdentifier from bag-info.txt
+          BagLocation(
+            storageNamespace = "uploadNamespace",
+            storagePrefix = Some("uploadPrefix"),
+            unpackBagRequest.storageSpace,
+            bagPath = BagPath("externalIdentifier")
```

Comment:
Where should we get the bagPath from?

Reply:
As per the comment, I think we need to read the bagInfo in the unpacker to get the external identifier (it would be nice to separate that functionality but I don't think it justifies another service).

Reply:
🤦‍♀️ 

Of course, I saw the comment about config and skipped straight past the second line.